assets/scss/projects.scss

```diff
@@ -28,6 +28,48 @@ html.modal-open, html.modal-open body {
     }
   }
 
+  .protocols-summary {
+    th:nth-child(7),
+    td:nth-child(7) {
```

Comment:
🚨 MAGIC NUMBERS 🚨 

Is this actually a `:last-child` or is there something else going on here? At the very least a comment to explain would be beneficial.

Reply:
yeah it sucks - but I couldn't find a way to select the rowspan item which occurs right after a non rowspan. I'll add a comment